### PR TITLE
Display error message for missing issuer; tx amount entry bugfix

### PR DESCRIPTION
**Pull request by burieberry**
CS-897
- Update boxel version
- Display required field error message on input or on blur for the Issuer Name field in Layout Customization card

CS-942
- Bugfix for invalid transaction amount entry into the input field (was causing uncaught error which stopped workflow from continuing)

req

To do:
- [x] Move `ember-unique-id-helper-polyfill` dependency in boxel from devDependencies to dependencies, and remove it from here
- [x] Tests
- [x] Check for invalid entry for issuer name field --> seems like only blank entry is invalid?

### CODE REVIEW

**Review by Aierie — CHANGES_REQUESTED** [review]

File: packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.hbs [hunk 1]
```diff
@@ -42,17 +42,22 @@
           @label="Amount to deposit"
           @fieldMode="edit"
           class="transaction-amount__field"
+          @labelClass="transaction-amount__input-field-label"
         >
         <div class="transaction-amount__input-container transaction-amount__input-token-balance">
           {{svg-jar this.currentTokenDetails.icon class="transaction-amount__currency-icon" role="presentation"}}
           {{!-- .boxel-input is a hack to get the boxel styles. --}}
-          <input
+          <Boxel::Input
```
[the remark this answers is not in the document]
Comment (Aierie) [reply packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.hbs:50]: https://app.uniswap.org/#/swap

[the remark this answers is not in the document]
Comment (Aierie) [reply packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.hbs:50]: To achieve this, I think we really should change Boxel::Input to use a native html input so we can get both input filtering and the validation errors showing.

[the remark this answers is not in the document]
Comment (Aierie) [reply packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.hbs:50]: https://github.com/cardstack/boxel/pull/375

[the remark this answers is not in the document]
Comment (burieberry) [reply packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.hbs:50]: I'll update this. It looks like Uniswap disables the CTA past 18 decimal places without explanation. For insufficient balance, they disable the CTA but have the insufficient balance notice shown on the button. 

We get an uncaught error past 18 decimal places and the button is not visually disabled, even though the unlocking process doesn't work. And there's no way to change the input once the button is pressed. I'll see if I can fix that quickly.

**Review by Aierie — APPROVED** [review]

**Pull request merged**